lib/prx_auth/rails/ext/controller.rb

```diff
@@ -44,6 +44,7 @@ def prx_authenticated?
 
       def authenticate!
         if !current_user
+          set_after_sign_in_path
```

Comment:
Previously, we were setting this session key on basically every request.  Which included a bunch of "turbo preloading" of links when you just hover over them!

Now, we just explicitly set it just before we redirect to the sessions controller.

Reply:
makes sense!